neuralop/layers/tests/test_finite_difference_conv.py

```diff
@@ -0,0 +1,72 @@
+####### ####### #######
+# This script discretizes a parabola on a 2d grid and check to see that 
+# the output converges to the theoretical limit.
+####### ####### #######
+
+import pytest
+import numpy as np
+import torch
+import math
+
+from ..differential_conv import FiniteDifferenceConvolution
+
+@pytest.mark.parametrize('implementation,resolution',
+                         [('subtract_middle', 500),
+                          ('subtract_middle', 700),
+                          ('subtract_middle', 1000),
+                          ('subtract_all', 500),
+                          ('subtract_all', 700),
+                          ('subtract_all', 1000)])
+def test_convergence_FiniteDifferenceConvolution(implementation, resolution):
+    def get_grid(S, batchsize, device):
+        gridx = torch.tensor(np.linspace(0, 1, S), dtype=torch.float)
+        gridx = gridx.reshape(1, 1, S, 1).repeat([batchsize, 1, 1, S])
+        gridy = torch.tensor(np.linspace(0, 1, S), dtype=torch.float)
+        gridy = gridy.reshape(1, 1, 1, S).repeat([batchsize, 1, S, 1])
+        return torch.cat((gridx, gridy), dim=1).to(device)
+    
+    torch.manual_seed(0)
+    device = torch.device('cpu')
+
+    num_channels = 10
+    kernel_size = 3
+    coeff = torch.rand((num_channels,))
+
+    differential_block = FiniteDifferenceConvolution(
+        in_channels=num_channels,
+        out_channels=1,
+        num_dim=2,
+        kernel_size=kernel_size,
+        groups=1,                       # mixing derivatives
+        implementation=implementation
+    ).to(device)
+
+    with torch.no_grad():
+        if implementation == 'subtract_middle':
+            weight = differential_block.conv.weight[0] # [0] because output channels is 1
```

Comment:
MIght be worse separating the unit-tests for each type (substract middle, all) so it is also easier to debug and add more later.